### ISSUE TITLE
ARIA: issue with Inline `#clr-color-marker`

### ISSUE THREAD

**Issue by melloware** [mdbassit/Coloris#133]
Using FastPass:

![image](https://github.com/mdbassit/Coloris/assets/4399574/e784b3c8-ac45-40ea-935f-bf27e8e0428a)

Looks like `#clr-color-marker` its saying there is no `role` attribute even though it uses `aria-label`.

Maybe it should be `role="presentation"`??

**Comment by jepsar** [comment]
@melloware Maybe we can ask @christophs78 to test the change.

**Comment by melloware** [comment]
@mdbassit you are right `role="presentation"` is not correct as the docs say 

> Note: The element with role="presentation" is not part of the accessibility tree and should not have an accessible name. Do not use [aria-labelledby](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-labelledby) or [aria-label](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-label).

**Comment by mdbassit** [comment]
Indeed, `role="presentation"` is the same as `role="none"`, so it shouldn't be applied to interactive elements.

I think for now we can safely ignore the warning from FastPass. I'm more concerned with how screen readers treat the UI elements than with strictly complying with the standards. I'm open to any suggestions though to improve compliance though.

**Comment by melloware** [comment]
Agreed should we leave this open for tracking and so no one else re-reports it?

**Comment by mdbassit** [comment]
Alright, let's leave it open for now
